src/p2p/behaviour.rs

```diff
@@ -16,16 +16,15 @@ use libp2p::mdns::{MdnsEvent, TokioMdns};
 use libp2p::ping::{Ping, PingEvent};
 use libp2p::swarm::toggle::Toggle;
 use libp2p::swarm::{NetworkBehaviour, NetworkBehaviourEventProcess};
-use libp2p::NetworkBehaviour;
 use multibase::Base;
 use std::{convert::TryInto, sync::Arc};
 use tokio::task;
 
 /// Behaviour type.
-#[derive(NetworkBehaviour)]
+#[derive(libp2p::NetworkBehaviour)]
```

Comment:
this was done as rust analyzer has this week started having issues with duplicate `use`'s, which I kind of hope will end up being the rustc behaviour in the future as I found this "two items for the same name" confusing as well.

Reply:
no downsides to doing it this way :+1: